docs/developer/extending/apps/subscription-webhook-payloads.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Subscription Webhook Payloads
+---
+
+## Custom payloads
+
+The whole idea behind subscription payloads for webhooks is simple. We want to be able to generate custom payloads for all webhooks using our GraphQL API. With subscriptions we are able to do that.
+Our Webhook objects have a `query` field that can hold subscription query converted to string. That query will be used to build a custom webhook payload.
```

Comment:
We should avoid saying "simple" in the docs ([see this article for instance](https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher)). I would rephrase this part a bit:

Webhook payloads in Saleor can be defined with GraphQL subscriptions. Subscription queries allow you to subscribe to different events, and for each one, define what fields should be returned in the payload.

Reply:
You're right. Changed to proposed.